redis/5.3/modules/ROOT/pages/redis-connector-reference.adoc

```diff
@@ -64,11 +63,11 @@ You can set configurations in Expression mode, except when Redis Connector is us
 ** SECONDS
 ** MINUTES
 ** HOURS
-** DAYS |  Time unit for Redis instance connection timeout. |  MILLISECONDS | 
-| Password a| String |  Redis password. Set this in case that the server requires authentication. |  | 
-| Entry TTL a| Number |  The default time to live to be set on keys stored through connector. |  | 
+** DAYS |  Time unit for Redis instance connection timeout. |  MILLISECONDS |
+| Password a| String |  Redis password. Set this in case that the server requires authentication. |  |
+| Entry TTL a| Number |  The default time to live to be set on keys stored through connector. |  |
 | Cluster Hosts a| String |  Redis cluster hosts separated by comma and having format host:port. Use this field when you want to access you Redis cluster through connector. |  | *x*
-| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment will fail if the test doesn't pass after exhausting the associated reconnection strategy |  | 
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment will fail if the test doesn't pass after exhausting the associated reconnection strategy |  |
```

Comment:
Question - do you need an article before 'deployment'? Also, do we say: 'fails' vs. 'will fail'?

Reply:
It is, but this is in existing material that doesn't pay to fix since it is all over the doc.